index.js

```diff
@@ -1,663 +1,246 @@
+const root = document.createElement('div');
 root.id = 'root';
 input.id = 'keyboard-input';
 document.body.append(input);
 document.body.append(root);
 input.disabled = true;
 
 // Creature DOM
-keyboard.forEach(function(row) {
-    const rowWraper = document.createElement('div');
-    row.forEach(function(buttonObj) {
-        const key = Object.keys(buttonObj)[0];
-        const button = document.createElement('button');
-        button.id = key;
-        button.setAttribute('uniqueAttribute', buttonObj[key][currentLang][0]);
-        button.classList.add('button');
-        button.innerHTML = buttonObj[key][currentLang][0];
-        rowWraper.appendChild(button);
-    });
-    root.appendChild(rowWraper);
+keyboard.forEach((row) => {
+  const rowWraper = document.createElement('div');
+  row.forEach((buttonObj) => {
+    const key = Object.keys(buttonObj)[0];
+    const button = document.createElement('button');
+    button.id = key;
+    button.setAttribute('uniqueAttribute', buttonObj[key][currentLang][0]);
+    button.classList.add('button');
+    const [lower] = buttonObj[key][currentLang];
+    button.innerHTML = lower;
+    rowWraper.appendChild(button);
+  });
+  root.appendChild(rowWraper);
 });
 
 // language switching
 function switchLang() {
-    currentLang === 'en' ? currentLang = 'ru' : currentLang = 'en';
-    localStorage.setItem('lang', currentLang);
+  if (currentLang === 'en') {
+    currentLang = 'ru';
+  } else {
+    currentLang = 'en';
+  }
+  localStorage.setItem('lang', currentLang);
 }
 
 // Lowercase
 function updateKeys() {
-    keyboard.forEach(function(row) {
-        row.forEach(function(buttonObj) {
-            const key = Object.keys(buttonObj)[0];
-            const button = document.getElementById(key);
-            button.setAttribute('uniqueAttribute', buttonObj[key][currentLang][0]);
-            button.innerHTML = buttonObj[key][currentLang][0];
-        })
-    })
+  keyboard.forEach((row) => {
+    row.forEach((buttonObj) => {
+      const key = Object.keys(buttonObj)[0];
+      const button = document.getElementById(key);
+      button.setAttribute('uniqueAttribute', buttonObj[key][currentLang][0]);
+      const [lower] = buttonObj[key][currentLang];
+      button.innerHTML = lower;
+    });
+  });
 }
 
 // Uppercase
 function shiftUpdateKeys() {
-    keyboard.forEach(function(row) {
-        row.forEach(function(buttonObj) {
-            const key = Object.keys(buttonObj)[0];
-            const button = document.getElementById(key);
-            button.setAttribute('uniqueAttribute', buttonObj[key][currentLang][1]);
-            button.innerHTML = buttonObj[key][currentLang][1];
-        })
-    })
+  keyboard.forEach((row) => {
+    row.forEach((buttonObj) => {
+      const key = Object.keys(buttonObj)[0];
+      const button = document.getElementById(key);
+      button.setAttribute('uniqueAttribute', buttonObj[key][currentLang][1]);
+      const [, upper] = buttonObj[key][currentLang];
```

Comment:
buttonObj[key][currentLang][1] and const [, upper] = buttonObj[key][currentLang]; are the same